tjackson2/lab4_test.py~

```diff
@@ -0,0 +1,9 @@
+'''09/28/2015 Tyler Jackson Testing Implementation for magic tests.

```

Comment:
The tilde files aren't showing up in the file explorer, I'm not sure where they keep coming from or how to delete them.